### PR TITLE
Tab-bars automatic hide and showing of content

### PR DESCRIPTION
**Pull request by rx**

### CODE REVIEW

File: views/mdc/assets/js/components/tab-bars.js [hunk 1]
```diff
@@ -10,5 +10,20 @@ export function initTabBars(e) {
 export class VTabBar extends eventHandlerMixin(VBaseComponent) {
     constructor(element, mdcComponent) {
         super(element, mdcComponent);
+
+        mdcComponent.listen('MDCTabBar:activated', (event) => {
+            console.log(event);
```
Comment (jadefish) [inline views/mdc/assets/js/components/tab-bars.js]: This should be `console.debug` or be removed.

File: views/mdc/assets/js/components/tab-bars.js [hunk 2]
```diff
@@ -10,5 +10,20 @@ export function initTabBars(e) {
...
+            const tabs = element.parentElement.querySelectorAll('.v-tab-content');
+
+            tabs.forEach(function(element) {
```
Comment (jadefish) [inline views/mdc/assets/js/components/tab-bars.js]: A for...of loop will be more performant and is (imo) more readable:
```suggestion
            for (const element of tabs) {
```

File: views/mdc/components/tab_bar.erb [hunk 3]
```diff
@@ -27,3 +27,9 @@
     </div>
   </div>
 </div>
+<% comp.tabs.each_with_index do |tab, index|
+  next unless tab.components.any? %>
```
Comment (jadefish) [inline views/mdc/components/tab_bar.erb]: Here, if we have a tab without any components, we'll be skipping an index as the `<div>` below would not be rendered. Will the JS handler for `MDCTabBar:activated` behave correctly if an index isn't found in the DOM?

Comment (rx) [reply views/mdc/components/tab_bar.erb]: It works just fine.

**Pull request merged**